.github/workflows/daily-test.yml

```diff
@@ -101,12 +101,6 @@ jobs:
         with:
           node-version: 18.x
```

Comment:
A separate observation: With node18 installed, shall we remove L60?

Reply:
no node 15 is needed to run the cma script. node 18 is needed to use CW SDK.